cpp/include/raft/random/make_regression.cuh

```diff
@@ -100,6 +102,82 @@ void make_regression(const raft::handle_t& handle,
                                  type);
 }
 
+/**
+ * @brief GPU-equivalent of sklearn.datasets.make_regression as documented at:
+ * https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_regression.html
+ *
+ * @tparam  DataT  Scalar type
+ * @tparam  IdxT   Index type
+ *
+ * @param[in]   handle          RAFT handle
+ * @param[out]  out             Row-major (samples, features) matrix to store
+ *                              the problem data
+ * @param[out]  values          Row-major (samples, targets) matrix to store
+ *                              the values for the regression problem
+ * @param[in]   n_informative   Number of informative features (non-zero
+ *                              coefficients)
+ * @param[out]  coef            If present, a row-major (features, targets) matrix
+ *                              to store the coefficients used to generate the values
+ *                              for the regression problem
+ * @param[in]   bias            A scalar that will be added to the values
+ * @param[in]   effective_rank  The approximate rank of the data matrix (used
+ *                              to create correlations in the data). -1 is the
+ *                              code to use well-conditioned data
+ * @param[in]   tail_strength   The relative importance of the fat noisy tail
+ *                              of the singular values profile if
+ *                              effective_rank is not -1
+ * @param[in]   noise           Standard deviation of the Gaussian noise
+ *                              applied to the output
+ * @param[in]   shuffle         Shuffle the samples and the features
+ * @param[in]   seed            Seed for the random number generator
+ * @param[in]   type            Random generator type
+ */
+template <typename DataT, typename IdxT>
+void make_regression(
+  const raft::handle_t& handle,
+  raft::device_matrix_view<DataT, raft::matrix_extent<IdxT>, raft::row_major> out,
```

Comment:
```suggestion
  raft::device_matrix_view<DataT, IdxT, raft::row_major> out,
```

Reply:
@divyegala Thanks!  I wonder why that even compiled....